docs/samples/Microsoft.ML.Samples/Dynamic/ModelOperations/OnnxConversion.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic.ModelOperations
+{
+    public static class OnnxConversion
+    {
+        private class ScoreValue
+        {
+            public float Score { get; set; }
+        }
+
+        private class OnnxScoreValue
+        {
+            public VBuffer<float> Score { get; set; }
+        }
+
+        private static void PrintScore(IEnumerable<ScoreValue> values, int numRows)
+        {
+            foreach (var value in values.Take(numRows))
+                Console.WriteLine("{0, -10} {1, -10}", "Score", value.Score);
+        }
+
+        private static void PrintScore(IEnumerable<OnnxScoreValue> values, int numRows)
+        {
+            foreach (var value in values.Take(numRows))
+                Console.WriteLine("{0, -10} {1, -10}", "Score", value.Score.GetItemOrDefault(0));
+        }
+
+        public static void Example()
+        {
```

Comment:
I think this sample is getting more complicated than necessary. Can we simplify it just to demonstrate basic Onnx export and exporting to a different opset? The scenarios of using the whole pipeline versus partial pipeline is more advanced and can be left out of this sample.

Reply:
Sure

---
In reply to: [436857134](https://github.com/dotnet/machinelearning/pull/5195#discussion_r436857134) [](ancestors = 436857134)